keps/2076-kueuectl/README.md

```diff
@@ -225,6 +226,18 @@ Output:
 * Position in Queue (if Pending)
 * Age
 
+### List ResourceFlavors
+
+Lists ResourceFlavors. Format:
+
+```
+kueuectl list resourceflavors 
+```
+
+Output:
+
+* Name
```

Comment:
uhm... maybe it's worth adding the node labels, but it can be a separate PR to update the CRD

Reply:
Follow up #2557